### DISCUSSION TITLE
建议有选项可以“傻瓜化一键安装”，不用选那么多选项

### DISCUSSION THREAD

**Discussion by flywhc** [jinwyp/one_click_script#181] [Ideas]
这个脚本我用了好几年，功能越来越强多，我也给周围朋友推荐。
但是对于不懂技术的人来说这么多选项也挺难抉择的，特别是有的选项还有先后顺序，也不确定是不是可以不重启就连装不同选项。

能否提供一个默认按回车就能把所有功能就都装好？
最多只需要输入必要的参数，如域名和端口，就把BBR、Netflix限制、xray等都装好。
或者能接受命令行参数也行，这样我只需要告诉朋友命令行时把各种参数都输好，不懂技术的朋友拷贝粘贴一行命令就好了。

**Comment by jinwyp** [comment]
这个脚本就不是针对简单傻瓜式的一键脚本.  否则也不会有那么多菜单。

稍后会增加一个菜单 只针对最简单的方式安装。

bbr 可以不安装。 Netflix也可以不用安装 只装xray。
